go.mod

```diff
@@ -11,3 +11,5 @@ require (
 	github.com/mattermost/mattermost-server/v6 v6.0.0-20220315170027-7710a2fe3741
 	github.com/pkg/errors v0.9.1
 )
+
+replace github.com/mattermost/mattermost-plugin-apps v0.7.1-0.20220114173732-f10616dc8752 => ../mattermost-plugin-apps
```

Comment:
remove?

Reply:
@levb I was waiting on https://github.com/mattermost/mattermost-plugin-apps/pull/304 to be merged so I can point to the merge commit. Do you suggest another path here?

Reply:
Fixed